Samples/Swift/DaysUntilBirthday/iOS/UserProfileView.swift

```diff
@@ -57,9 +57,7 @@ struct UserProfileView: View {
         .toolbar {
           ToolbarItemGroup(placement: .navigationBarTrailing) {
             Button(NSLocalizedString("Disconnect", comment: "Disconnect button"), action: disconnect)
-              .accessibilityLabel(Text("Disconnect scope button."))
```

Comment:
What were these here for?

Reply:
I was originally overzealous with adding `accessibilityLabel(_:)` before. It's useful for labeling a view without text, but since this is a button with a clear label, we don't need it.

[Documentation](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-1d7jv)